src/styles/lists.styl

```diff
@@ -62,21 +62,46 @@ heavyShadow = 0 4px 24px 0 rgba(50, 54, 63, .08)
     display grid
     grid-template-columns repeat(auto-fit, serviceTileSize)
     grid-auto-rows serviceTileSize
-    grid-gap rem(16)
+    grid-gap rem(17)
     justify-content center
+    opacity .99 // required for the blend mode difference to work
 
     +tiny-screen()
         margin-top rem(24)
         grid-template-columns repeat(auto-fit, mobileTileSize)
         grid-auto-rows mobileTileSize
-        grid-gap rem(2)
+        grid-gap rem(3)
 
     .item
         padding 12px 8px
         border-radius 8px
         border-width 1px
         border-style solid
         border-color transparent
+        position relative
+
+        &:before,
+        &:after
+            content ''
+            display block
+            position absolute
+            top 30%
+            bottom 30%
+            width 1px
+            opacity .32
+            mix-blend-mode difference
```

Comment:
Clever!

Reply:
All credits go to @joel-costa for this one !